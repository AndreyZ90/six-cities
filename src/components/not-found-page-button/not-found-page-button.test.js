import React from 'react';
import TestRenderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';

import NotFoundPageButton from '@/components/not-found-page-button/not-found-page-button';

describe(`NotFoundPageButton component snapshot`, () => {
  test(`Should correctly render NotFoundPageButton component`, () => {
    const tree = TestRenderer.create(
        <MemoryRouter>
          <NotFoundPageButton />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
