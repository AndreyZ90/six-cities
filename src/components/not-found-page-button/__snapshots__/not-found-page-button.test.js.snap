// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPageButton component snapshot Should correctly render NotFoundPageButton component 1`] = `
<button
  className="page-not-found__button"
>
  <a
    href="/"
    onClick={[Function]}
  >
    go to the main page
  </a>
</button>
`;
